scripts/createWebservers.sh

```diff
@@ -104,55 +105,31 @@ SetupNginx() {
 
 SetupWebApps() {
     LogInfo "Setting up .NET web service"
-    current_dir=$(pwd)
 
-    cd sampleWebService
-
-    $ctr_cli build -t samplewebservice . > webService.log
+    $ctr_cli image pull mliddle2/tunnelwebapp > webService.log
 
     $ctr_cli run -d \
-        --name=webApp \
+        --name=webapp \
         --restart=unless-stopped \
         -p 8081:80 \
         -p 8443:443 \
+        -e PROXY_IP=$PROXY_IP \
         -e ASPNETCORE_URLS="https://+;http://+" \
         -e ASPNETCORE_HTTPS_PORT=443 \
         -e ASPNETCORE_Kestrel__Certificates__Default__Password="" \
         -e ASPNETCORE_Kestrel__Certificates__Default__Path=/https/server.pfx \
         -v /etc/pki/tls/private:/https/ \
-        samplewebservice >> webService.log 2>&1
-
-    $ctr_cli run -d \
-        --name=excluded \
-        --restart=unless-stopped \
-        -p 8080:80 \
-        -p 9443:443 \
-        -e ASPNETCORE_URLS="https://+;http://+" \
-        -e ASPNETCORE_HTTPS_PORT=443 \
-        -e ASPNETCORE_Kestrel__Certificates__Default__Password="" \
-        -e ASPNETCORE_Kestrel__Certificates__Default__Path=/https/server.pfx \
-        -v /etc/pki/tls/private:/https/ \
-        samplewebservice >> webService.log 2>&1
-
-    cd $current_dir
+        mliddle2/tunnelwebapp >> webService.log 2>&1
```

Comment:
You don't want this right?

Reply:
mliddle2 is the user that needs to be used in the path for that particular image